src/sentry/static/sentry/app/views/settings/components/forms/rangeField.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import InputField from 'app/views/settings/components/forms/inputField';
+import RangeSlider from 'app/views/settings/components/forms/controls/rangeSlider';
+
+type DefaultProps = {
+  formatMessageValue?: false | Function;
+};
+
+type DisabledFunction = (props: any) => boolean;
+type PlaceholderFunction = () => string;
+
+type Props = DefaultProps &
+  Omit<RangeSlider['props'], 'value' | 'disabled' | 'placeholder'> &
+  Omit<InputField['props'], 'field'> & {
+    disabled?: boolean | DisabledFunction;
+    placeholder?: string | PlaceholderFunction;
+  };
+
+function onChange(fieldOnChange, value, e) {
+  fieldOnChange(value, e);
+}
+
+function defaultFormatMessageValue(value, props: Props) {
+  return (typeof props.formatLabel === 'function' && props.formatLabel(value)) || value;
+}
+
+export default function RangeField({
+  formatMessageValue = defaultFormatMessageValue,
+  ...otherProps
+}: Props) {
+  if (typeof otherProps.disabled === 'function') {
+    otherProps.disabled = otherProps.disabled(otherProps);
```

Comment:
This is due to a mismatch between what `RangeSlider` accepts and what is provided from `fieldFromConfig` and upstream.